File: Web/Eye/Eye/Model/Workspace.cs
namespace EYE.Model
{
    public class Workspace
    {

        private int codWorkspace;
        private string workspacename;
        private string nome;
        private string email;
        private string senha;
        private int salt;

		public int CodWorkspace { get => codWorkspace; set => codWorkspace = value; }
		public string Workspacename { get => workspacename; set => workspacename = value; }
		public string Nome { get => nome; set => nome = value; }
		public string Email { get => email; set => email = value; }
		public string Senha { get => senha; set => senha = value; }
		public int Salt { get => salt; set => salt = value; }
	}
}

File: Local/Eye/src/main/java/br/com/eye/model/Workspace.java
package br.com.eye.model;


public class Workspace {

    private int codWorkspace;

    public int getCodWorkspace() {
        return codWorkspace;
    }

    public void setCodWorkspace(int codWorkspace) {
        this.codWorkspace = codWorkspace;
    }

<<<<<<< HEAD
=======
    public boolean ValidarWorkspace(String workspacename) throws SQLException {
        return new ControllerWorkspace().WorkspaceValido(workspacename);
    }
>>>>>>> 4287f4d4939e422c23471e5bd48b4e22fd8b88fe

}
